backend/app/routers/auth.py

```
@@ -46,61 +68,71 @@
 
 @router.post("/reset-password")
-async def reset_password(request_data: ResetPasswordRequest, db: Session = Depends(get_db)):
+async def reset_password(
+    request_data: ResetPasswordRequest, db: Session = Depends(get_db)
+):
     email = verify_reset_token(request_data.token)
     if email is None:
-        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="Invalid or expired token")
-    
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST, detail="Invalid or expired token"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)